buildSrc/src/main/groovy/bio.terra.catalog.java-shared-conventions.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'org.liquibase.gradle'
+}
+dependencies {
```

Comment:
This is OK although I was trying to keep the conventions plugins as a tree so you'd only use on per subproject. But I guess there's no real reason to require it to be that way. I think it can get confusing when you see different plugins used and it's harder to know what the combination of all of them might do.

Reply:
Yeah that makes sense I was just trying to refactor shared dependencies, but it isn't necessary. It might be clearer if I revert this change. Do you have an opinion on that?